examples/ts-ecr-dockerfile/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+
+const repository = new awsx.ecr.Repository("repository", { forceDelete: true });
+
+export const image = new awsx.ecr.Image("image", {
+  repositoryUrl: repository.repository.repositoryUrl,
+  context: "./app",
+  dockerfile: "./docker/Dockerfile",
```

Comment:
How does the test fail without your change? Can't find Dockerfile?

Reply:
It [fails](https://github.com/pulumi/pulumi-awsx/actions/runs/6632554026/job/18018645089?pr=1115#step:15:680) with

> error: could not open dockerfile at relative path app/Dockerfile: stat app/Dockerfile: no such file or directory